roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -29,6 +29,17 @@
   failed_when: "'already exists' not in osmiq_create_service_account.stderr and osmiq_create_service_account.rc != 0"
   changed_when: osmiq_create_service_account.rc == 0
 
+- name: Create Image Scanner Service Account
```

Comment:
I don't like much "image scanner" ...after all... why inventing another term (like "scanner") when we already have "inspector".

Although I am not sure I like `management-image-inspector`... any other suggestion? @enoodle @detiber @deads2k